libs/util/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@cordis/util",
+  "version": "0.1.0",
+  "description": "Helper REST functions and other utilities for Cordis",
+  "main": "./dist/index.js",
+  "types": "./types/index.d.ts",
+  "scripts": {
+    "lint": "eslint src --ext .ts",
+    "build": "tsc"
+  },
+  "files": [
+    "dist",
+    "types",
+    "!dist/*.tsbuildinfo",
+    "!**/**.map",
+    "!**/*.test.*"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cordis-lib/cordis.git"
+  },
+  "contributors": [
+    "didinele <didinele.dev@gmail.com> (https://github.com/didinele)",
+    "Holo-Buckshot (https://github.com/Holo-Buckshot)",
+    "Nico (https://github.com/zaida04)"
+  ],
+  "author": "Zaid \"Nico\"",
+  "license": "MIT",
```

Comment:
Uh, code base is licensed under Apache 2. Should exclude this field anyway, pointless to have it everywhere.